edxlearndot/management/commands/update_learndot_enrolments.py

```diff
@@ -94,15 +107,11 @@ def handle(self, *args, **options):
             for enrollment in enrollments:
                 contact_id = learndot_client.get_contact_id(enrollment.user)
                 if not contact_id:
-                    log.error("Could not locate Learndot contact for user %s", enrollment.user)
-                    continue
-
-                enrolment_id = learndot_client.get_enrolment_id(contact_id, cm.learndot_component_id)
-
-                if not enrolment_id:
-                    log.error("No enrolment found for contact %s, component %s", contact_id, cm.learndot_component_id)
-                    continue
-
+                    log.info(
+                        "Not setting enrolment status for user %s in course %s, because contact_id is None.",
+                        enrollment.user,
+                        cm.edx_course_key
+                    )
```

Comment:
@rocioar Needs a `continue` here, so we skip to the next enrollment.

Reply:
my bad there, fixed it.